### PR TITLE
add html about mike

### PR DESCRIPTION
**Pull request by mikemangas**
mikes personal website

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1 +1,98 @@
...
+  content="width=device-width, initial-scale=1.0, user-scalable=no"
+  charset="UTF-8"
+/>
+<html>
+  <head>
+    <title>Mike´s Personal Page</title>
+  </head>
+  <body>
+    <header>
+      <h1>Michail Mangasarov - the next Bill Gates</h1>
```
Comment (mpagels) [inline index.html:13]: ;-)

File: index.html [hunk 2]
```diff
@@ -1 +1,98 @@
...
+      <nav>
+        <ul>
+          <li><a href="#contact">Contact</a></li>
+          <li><a href="#bio">Bio</a></li>
+        </ul>
+      </nav>
```
Comment (mpagels) [inline index.html:19]: Very good.

File: index.html [hunk 3]
```diff
@@ -1 +1,98 @@
...
+        <li><a href="https://kindersport-wissen.de">kindersport wissen</a> - Blog owner </li>
+        <li><a href="https://mangasmedia.de">mangasmedia</a> - Web services & photography for Restaurants</li>
+        <li>Recognified, Munich – Account Manager</li>
+        <li>AdAudience, Munich – Junior Campaign & Programmatic Manager</li>
+        <li>International Olympic Academy, Olympia, Greece – Coordinator</li>
+        <li>GUtech, Muscat, Oman – Projekt Manager in University Sports</li>
+        <li>Restaurant Thalassa, Munich – Marketing Manager</li>
+        <li>DIE WELT, Berlin – Internship as a Sports Journalist</li>
+      </ul>
+    </section>
```
Comment (mpagels) [inline index.html:87]: I would recommend putting your main content into a `main` element.

File: index.html [hunk 4]
```diff
@@ -1 +1,98 @@
...
+        <li>CSS</li>
+        <li>Javascript</li>
+      </ul>
 
+      <h2>My Interests</h2>
+      <ul>
+        <li>Boxing</a></li>
+        <li>Football</li>
+        <li>Working with digital Media</li>
+      </ul>
```
Comment (mpagels) [inline index.html:45]: Maybe this could be a section on its own?

File: index.html [hunk 5]
```diff
@@ -1 +1,98 @@
...
+    <section id="contact">
+      <address>
+        <h2>Get in touch with me</h2>
+          <a href="tel:+4917656763760">0176 567 637 60</a><br>
+          <a href="mailto:mangasarov@hotmail.de">mangasarov{@}hotmail.de</a>
+      </address>
+    
+    </section>
+    <section id="socials">
+      <h3>Feel free to follow me on my socials</h3>
```
Comment (mpagels) [inline index.html:56]: Try not to skip headlines. Read more about it here:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#usage_notes

File: index.html [hunk 6]
```diff
@@ -1 +1,98 @@
...
+<h2>About me</h2>
+<p>Hello, my name is Michail Mangasarov, but everyone calls me Mike. I am really happy, that you visited my personal Site.
+  I can call myself an alrounder, not only in the area of digital media, but also in life.
+  <br>
+  While I really like to coach children in sports, 
+  i also like to take challenges. This year i build a small house in Cyprus - no engineer or architect. Just using the internet as my source of knowledge.
+  <br>Furthermore I really like photography. I like to take photographs of people. Especially from my 1 year old daugther, but also from various cultures from other countries.
+<br>
+  I am an open, communicative person, who wants to be around positive people in an positive working atmosphere, where everyone appreciates and respects the others works and commitments.
+</p>
```
Comment (mpagels) [inline index.html:70]: Did you install `prettier`? This looks a bit moved.
https://prettier.io/

**Review by mpagels — CHANGES_REQUESTED** [review]
Very good mike. Found some small things. And a question regarding prettier. Pls review my review.

File: index.html (hunk 6 above)
Comment (mikemangas) [reply index.html:70]: i did, but unfortunately it doesnt work properly.. dont really know how to fix it. but i will check it out, when the class today is over.

File: index.html (hunk 3 above)
Comment (mikemangas) [reply index.html:87]: great, thank you for that information! would you consider my CV as my main information?

Comment (mpagels) [reply index.html:87]: You have to think semantically. When you put something into the main element, a screen reader would know, okay, everything in here is the main content. So very important. ;-)

File: index.html (hunk 6 above)
Comment (mpagels) [reply index.html:70]: If prettier has an issue, then something in the code is broken. You should go carefully through the code. Maybe a closing tag is missing. You can inspect the terminal to check for problems.

Bildschirmfoto 2021-06-17 um 15 24 15

File: index.html (hunk 3 above)
Comment (mikemangas) [reply index.html:87]: ok great! thank you!

**Pull request merged**